### PR TITLE
Add the Studio Explorer boilerplate HTML

### LINKED ISSUE #68: Studio Explorer Boilerplate HTML
## Requirements

This *replaces* the behavior of redirecting directly to studio with the more complete implementation which includes a landing-page that is served locally and _offers_ a redirect to Studio.  This will match the behavior of Apollo Server and Apollo Gateway today, exactly.  This offers more transparency to the user to understand what about to happen (the redirect) and allows them to optionally make the behavior sticky (on account of a browser cookie) for future requests. 

* Rather than redirect on the Router's configured endpoint, this replaces that redirect with serving of HTML **boilerplate** (e.g., with `content-type: text/html`) when the appropriate `accept` header with a satisfying `text/html` value is met.
* Renders the same boilerplate HTML that Apollo Server uses, [which the implementation for should be found here](https://github.com/apollographql/apollo-server/blob/a2fead3256b72a98588361c3702040162359cb42/packages/apollo-server-core/src/plugin/landingPage/default/index.ts#L1-L167)

Effectively, this should produce this experience: 
image

Will Resolve #380

### PR DESCRIPTION
**Pull request by bnjjj**
close #68

### CODE REVIEW

**Review by Geal — APPROVED** [review]

File: apollo-router/resources/index.html
Comment (abernix) [inline apollo-router/resources/index.html:15]:
```suggestion
    <title>Apollo Router</title>
```

File: apollo-router/resources/index.html
Comment (abernix) [inline apollo-router/resources/index.html:11]:
```suggestion
    <meta name="description" content="Apollo Router landing page" />
```

File: apollo-router/resources/index.html
Comment (abernix) [inline apollo-router/resources/index.html:41]:
```suggestion
            <h1>Welcome to the Apollo Router</h1>
```

File: apollo-router/resources/index.html
Comment (abernix) [inline apollo-router/resources/index.html:47]: We ... need to add this to our list of supported introspection queries. 🤣   (@o0Ignition0o can help with that if you're curious.)

**Review by abernix — APPROVED** [review]
Approving with one potentially important consideration that should be resolved before merging around the registered introspection query, and a couple text tweaks, but I can confirm that this looks and works great otherwise!  Nice job!

File: apollo-router/resources/index.html
Comment (bnjjj) [reply apollo-router/resources/index.html:47]: I added a new supported introspection query. I hope it's the right place @o0Ignition0o

Comment (o0Ignition0o) [reply apollo-router/resources/index.html:47]: Ok I have good and bad news for you, good news is you added it in the right place, the bad news is ... we're matching exact queries so you want to "save without formatting" (you have `\n` s afaict)

Comment (bnjjj) [reply apollo-router/resources/index.html:47]: Should be good now

**Pull request merged**
